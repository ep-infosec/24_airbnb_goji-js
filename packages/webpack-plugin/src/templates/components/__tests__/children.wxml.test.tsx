import { renderTemplate } from '../..';
import { childrenWxml } from '../children.wxml';

describe('children.wxml', () => {
  test('works', () => {
    expect(
      renderTemplate({ target: 'wechat', nodeEnv: 'development' }, () =>
        childrenWxml({
          relativePathToBridge: '.',
          componentDepth: 1,
        }),
      ),
    ).toMatchSnapshot();
  });
});
