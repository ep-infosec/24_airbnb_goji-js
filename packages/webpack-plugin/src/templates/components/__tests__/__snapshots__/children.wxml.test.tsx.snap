// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`children.wxml works 1`] = `
"<import src=\\"./components1.wxml\\" />
<block wx:for=\\"{{meta.children}}\\" wx:key=\\"gojiId\\">
  <template is=\\"$$GOJI_COMPONENT1\\" data=\\"{{ meta: item }}\\" />
</block>"
`;
